apps/L1E/ticket-booking/ticket-booking-frontend/specs/components/TicketAddDialog.test.tsx

```
@@ -0,0 +1,287 @@
+        type="text"
+        placeholder={placeholderText}
+        className={className}
+        value={selected ? selected.toISOString().split('T')[0] : ''}
+        onChange={(e) => {
+          // For testing purposes, just call onChange with a mock date
+          // The actual value will be set by the input's value prop
+          onChange(new Date('2024-12-25'));
+        }}
+      />
+    );
+  };
```

ESLint

[error] Function component is not an arrow function

(react/function-component-definition)